scene/main/http_request.cpp

```diff
@@ -349,6 +349,8 @@ bool HTTPRequest::_update_connection() {
 			}
 
 			client->poll();
+			if (client->get_status() != HTTPClient::STATUS_BODY)
+				break; // State changed after this poll, will check at next iteration.
```

Comment:
Needs a style fix to match the guidelines of the `master` branch:
```suggestion
			if (client->get_status() != HTTPClient::STATUS_BODY) {
				break; // State changed after this poll, will check at next iteration.
			}
```

But also needs a review from @Faless to see if this is intended to be merged or not.